src/main/webapp/script.js

```diff
@@ -73,6 +78,7 @@ function drawRegionsMap(censusDataArray) {
     'resolution': 'provinces',
     'colorAxis': {colors: ['white', 'blue']},
   };
+  document.getElementById('map-title').innerHTML = title;
```

Comment:
Is there a reason to use innerHTML instead of innertText, since the title is just regular text?

Reply:
Switched to innerText